public/js/admin.js

```diff
@@ -1,7 +1,7 @@
 /* This is the index for referencing colours for the snackbars */
 var colours = Object.freeze({
-    SUCCESS_GREEN: '#4BB543',
-    FAIL_RED: '#D93232',
+    SUCCESS_GREEN: 'green',
+    FAIL_RED: 'red darken-4',
```

Comment:
Can we add a warning_yellow, might be useful in the future. For example we should warn the prof that the question is now hidden.

Reply:
Drop the colors at the end of the names. Success_green > success and so on

Reply:
Done, gonna re-push.